### PR TITLE
Spanish tutorials import from Crowdin

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Imports Spanish `es` with latest from Crowdin for **Tutorials I** content bucket

## Related Issue
Ongoing translation efforts

### CODE REVIEW

**Comment by minimalsm** [comment]
Build failed

**Comment by wackerow** [comment]
Appears to have been a timeout... Updated to `dev`, checks rerunning

**Comment by wackerow** [comment]
Another anchor tag parsing bug. Building without any errors locally, that should do it.

File: src/content/translations/es/developers/tutorials/calling-a-smart-contract-from-javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+author: jdourlens
+tags:
+  - "transacciones"
+  - "frontend"
+  - "JavaScript"
+  - "web3.js"
+skill: principiante
+lang: es
+sidebar: true
+published: 19/04/2020
```
Comment (minimalsm) [inline src/content/translations/es/developers/tutorials/calling-a-smart-contract-from-javascript/index.md:13]: Dates are wrong on these

**Review by minimalsm — APPROVED** [review]

**Pull request merged**